handlers/pools.go

```diff
@@ -47,7 +49,7 @@ func Pools(w http.ResponseWriter, r *http.Request) {
 
 	data.Data = poolData
 
-	if handleTemplateError(w, r, poolsServicesTemplate.ExecuteTemplate(w, "layout", data)) != nil {
+	if handleTemplateError(w, r, "pools.go / Pools", poolsServicesTemplate.ExecuteTemplate(w, "layout", data)) != nil {
```

Comment:
This should be changed to
`if handleTemplateError(w, r, "pools.go / Pools / Done", ...`
to be consistent with your other changes.